taskchampion/CONTRIBUTING.md

```diff
@@ -39,13 +36,13 @@ Aside from that, start reading the docs and the source to learn more!
 The book documentation explains lots of the concepts in the design of TaskChampion.
 It is linked from the README.
 
-There are three crates in this repository.
+There are three important crates in this repository.
 You may be able to limit the scope of what you need to understand to just one crate.
  * `taskchampion` is the core functionality of the application, implemented as a library
- * `taskchampion-cli` implements the command-line interface (in `cli/`)
- * `taskchampion-sync-server` implements the synchronization server (in `sync-server/`)
+ * `taskchampion-lib` implements a C API for `taskchampion`, used by Taskwarrior
+ * `integration-tests` contains some tests for integrations between multiple crtes.
```

Comment:
```suggestion
 * `integration-tests` contains some tests for integrations between multiple crates.
```

RFC: Install Codespell in @djmitche's Editor :)

Reply:
I really should!